slurm_ops_manager/slurm_ops.py

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python3
```

Comment:
Is the shebang line needed?

Reply:
I don't think so. Will remove it and check.